spec/lambdaSpec.js

```diff
@@ -33,6 +33,7 @@ const mockMultiRequest = (config) =>
     if (config[url]) {
       return mockRequest(config[url])(opts, cb);
     } else {
+      console.error(`No mock found for request ${url}`);
```

Comment:
The problem was here - if the test do not have a mock for a specific URL, `fail` is invoked to fail the test. However, this is asynchronous and not obvious. Also, it seems to be falling back to the 'real' `require` module as well.

I've added a `console.error` so that you are warned immediately of the lack of a mock.